server/php/UploadHandler.php

```diff
@@ -1355,7 +1366,8 @@ public function head() {
 
     public function get($print_response = true) {
         if ($print_response && $this->get_query_param('download')) {
-            return $this->download();
+            $this->download();
+            return null;
```

Comment:
The `download` function already returns `null`.

Reply:
download() does not return anything, so it's a void, which cannot be returned.
Admittedly I was a bit unsure on what to return here instead, I figured a null would be the best, but I have to trace where the returned value of get() is being used, to see if it will be tested against being null.